surefire-its/src/test/java/org/apache/maven/surefire/its/RunOrderIT.java

```diff
@@ -97,7 +97,17 @@ public void testRandomJUnit4SameSeed()
             }
         }
     }
-    
+
+    @Test
+    public void testRandomJUnit4PrintSeed()
+    {
+        long seed = 0L;
+        OutputValidator validator = executeWithRandomOrder( "junit4", seed );
+        validator.verifyTextInLog( "To reproduce ordering use flag" );
+        validator = executeWithRandomOrder( "junit4" );
+        validator.verifyTextInLog( "To reproduce ordering use flag" );
```

Comment:
please consider to split into two test, eg:
- testRandomJUnit4PrintSeedWithGivenSeed
- testRandomJUnit4PrintSeedWithNoGivenSeed